### PR TITLE
Added custom css page

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/1626

### CODE REVIEW

File: bundles/admin/css.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Include custom css
+==================
+
+Sulu provides the package require-css_ to manage custom css. you can load
```
Comment (danrot) [inline bundles/admin/css.rst]: `custom css. You`

File: bundles/admin/css.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+css files in any require component with the prefix `css!`. You can omit the
```
Comment (danrot) [inline bundles/admin/css.rst]: `You have to omit`, i think that's not optional, isn't it?

File: bundles/admin/css.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+file extension.
+
+.. code-block:: javascript
+
+    define(['css!suluadmin/css/main'], function(){});
```
Comment (danrot) [inline bundles/admin/css.rst]: Don't you want to add a more realistic entry? I thought you had to add a path like `suluadmincss`

File: bundles/admin/css.rst (hunk 2 above)
Comment (wachterjohannes) [inline bundles/admin/css.rst]: your totally right (:

**Comment by wachterjohannes** [comment]
@danrot fixed (:

**Pull request merged**